projects/plugins/jetpack/extensions/blocks/donations/amount/save.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+const Save = () => <InnerBlocks.Content />;
```

Comment:
The Amount block doesn't have `InnerBlocks`, so this Save wouldn't save any HTML in the post content.
We might as well remove it, and replace it with a simple `save: () => null` in the block definition.

Reply:
Updated to your suggestion.